src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperationTest.java

```diff
@@ -2467,6 +2467,132 @@ public void findAllSortDescending() throws Exception {
     }
   }
 
+  @Test
+  public void findAllSortByUUIDv6() throws Exception {
+    // same for uuidv7
+    String collectionReadCql =
+        "SELECT key, tx_id, doc_json, query_text_values['uuidv6'], query_dbl_values['uuidv6'], query_bool_values['uuidv6'], query_null_values['uuidv6'], query_timestamp_values['uuidv6'] FROM \"%s\".\"%s\" LIMIT %s"
+            .formatted(KEYSPACE_NAME, COLLECTION_NAME, 20);
+
+    // These are UUIDv6 ids generated at 30 second intervals.
+    String doc1 =
+        """
+                      {
+                        "_id": "doc1",
+                        "uuidv6": "1e712685-714f-6720-a23a-c90103f70be6"
```

Comment:
What happens if you try to sort documents with mixed types, like they have a property `myId` and some documents have `myId` as a `uuidv6` and some have `myId` as a `uuidv7`?

Reply:
So, what we did for sort is just lexicographic by String representation. 
If there is a mix of uuidv6 and uuidv7, still sort in lexicographic order.
If I understand correctly, the timestamp design for v6 and v7 are different, then the result will not necessarily be in time order.